main/modes/jukebox/jukebox.h

```diff
@@ -12,4 +12,6 @@
 
 extern swadgeMode_t jukeboxMode;
 
+extern const char jukeboxName[];
```

Comment:
Love the consistency, but really?

Reply:
I'm pretty sure that was an artifact of me working on it at 2am lol